viadot/sources/base.py

```diff
@@ -124,6 +124,47 @@ def to_excel(
         out_df.to_excel(path, index=False, encoding="utf8")
         return True
 
+    def to_parquet(
+        self,
+        path: str,
+        if_exists: Literal["append", "replace", "skip"] = "replace",
+        if_empty: str = "warn",
+        **kwargs,
+    ) -> None:
+        """
+        Write from source to a Parquet file.
+
+        Args:
+            path (str): The destination path.
+            if_exists (str, optional): What to do if the file exists. Defaults to "replace".
+            if_empty (str, optional): What to do if the source contains no data. Defaults to "warn".
```

Comment:
Does it make sense to add this parameter if we only have 1 option?

Reply:
There are other options available, I updated docstring to make it clear